src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -25,4 +30,51 @@ public void equals_False_X10() {
     Integer i = Integer.valueOf(10);
     assertNotEquals(rnX, i);
   }
+
+  @Test
+  public void equals_True_XV_15() {
+    Integer i = Integer.valueOf(15);
+    assertTrue(i.equals(RomanNumeral.convertFromString("XV")));
+  }
+
+  @Test
+  public void equals_False_XV_10() {
```

Comment:
I think one Integer test is sufficient, but if you want to do more, use ParameterizedTest.